clis/teliod/src/config.rs

```diff
@@ -50,9 +52,21 @@ fn reconnect_after_expiry_default() -> Percentage {
     Percentage(90)
 }
 
-#[derive(PartialEq, Eq, Deserialize, Debug)]
+#[derive(Deserialize, Debug, Default)]
+pub struct ClientConfig {
+    pub hw_identifier: String,
```

Comment:
why would this be configurable? I think on the apps this was calculated as some hash from the UDID,  maybe we can use [something like this](https://crates.io/crates/mid) or how was this set for in tcli?

Reply:
tcli just joins publickey.name as the hw_identifier and I had used to same logic for this.

Reply:
So the user doesn't have to provide a `hw_identifier` to register their device? Then maybe it shouldn't be part of the config file?

Reply:
Removed.